bleak/backends/p4android/defs.py

```diff
@@ -87,4 +88,4 @@ class ScanFailed(enum.IntEnum):
     BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE: "write-without-response",
 }
 
-CLIENT_CHARACTERISTIC_CONFIGURATION_UUID = "2902"
+CLIENT_CHARACTERISTIC_CONFIGURATION_UUID = uuidstr_to_str(normalize_uuid_16(0x2902))
```

Comment:
```suggestion
CLIENT_CHARACTERISTIC_CONFIGURATION_UUID = normalize_uuid_16(0x2902)
```
`uuidstr_to_str()` changes the UUID to a human readable description, so I don't think this would work.

Reply:
Yeah.. I just got confused